packages/design-system/src/components/ChoiceList/Choice.tsx

```diff
@@ -110,16 +110,12 @@ export const Choice = (props: ChoiceProps) => {
   const { hintId, hintElement } = useHint({ ...props, id });
   const labelProps = useLabelProps({ ...props, id });
 
-  let errorId;
-  let errorElement;
-  if (props.errorMessage) {
-    errorId = props.errorId ?? `${props.id}__error`;
-    errorElement = (
-      <InlineError id={errorId} inversed={props.inversed} className={props.errorMessageClassName}>
-        {props.errorMessage}
-      </InlineError>
-    );
-  }
+  const errorId = props.errorId ?? `${props.id}__error`;
+  const errorElement = (
+    <InlineError id={errorId} inversed={props.inversed} className={props.errorMessageClassName}>
```

Comment:
I have a question around how our Choice/ChoiceList should handle this change.

Choice is used by ChoiceList. Individual Choices have their own InlineError. ChoiceList has its own InlineError (using `useInlineError`).

If I do what's asked (always show the live/atomic regions) there are at least two live/atomic regions shown (one for ChoiceList and one for Choice). I'm 99% sure if a Choice is used in ChoiceList, it shouldn't display a live/atomic region but I'm unsure how a child element would be aware if its being used in ChoiceList to know not to display some code.

I believe this logic is also screwing up some snap tests (seeing undefined IDs in 🚾).

Reply:
I think if we were to implement any kind of logic that took into account the context, it should go in another PR as an enhancement. The individual choice's inline error is really only supposed to be used for lone checkboxes, but we don't currently restrict it. I don't think it's a valid design to put an error message on a choice inside a choice list, so it's not something we _should_ be seeing in production.

Reply:
And also I think that enhancement would be to start blocking the use of choice inline error messages in choice list

Reply:
I think the future solve for this will be the following (in a follow-up ticket):

1. Update the Choice/ChoiceList pattern so individual Choice cannot show error messages when used in ChoiceList
2. ChoiceList passes an internal prop to Choice to hide the InlineError markup for individual Choices
3. ChoiceList throws warning if an error is detected in use of ChoiceList Choices